lessons/codap-exploring/langs/en-us/pages/codap-exploration.adoc

```diff
@@ -1,50 +1,100 @@
 = CODAP Exploration
+++++
+<style>
+body.workbookpage .autonum { padding-top: 0.5rem; }
+body.workbookpage .fitb { padding-top: 0.7rem;}
+</style>
+++++
 
-Make sure you’re logged into the @starter-file{animals} in CODAP. The purpose of this exploration is to familiarize yourself with some of CODAP's features. Feel free to experiment and try things out!
+This page will help you familiarize yourself with some of CODAP's features. Check off each item once you have completed it. Feel free to experiment and try things out! Make sure you’re logged into the @starter-file{animals} in CODAP before beginning.
 
-@n See if you can make a graph. What did you select in order to do so?
+== Tables in CODAP
 
-@fitb{}{@ifsoln{I clicked on the "Graph" icon in the upper left-hand corner of my screen}}
+@n A table of data is shown here, with the title at the top of the table. What is the title? @fitb{}{@ifsoln{Animals-Dataset-1.5.1}}
 
-@fitb{}{}
+@n Move the table to a different location on the screen.
 
-@vspace{2ex}
+@n Minimize the table. (Hint: Try hovering your mouse over the table's title. What appears?)
 
-@n Move the table. Move the graph. How do you move things in CODAP?
+@n Now re-expand the minimized table.
 
-@fitb{}{@ifsoln{To move tables and graphs, I mouse over the top-most bar until my cursor turns into a hand. Then I can drag.}}
+@n Add a row to the table, then delete that same row.
```

Comment:
this was tricky for me... I'd suggest offering a hint... I just needed to start typing and then hit return